client/src/ui/base/_themes.scss

```diff
@@ -294,18 +295,22 @@
   --curriculum-landing-started-advanced: url("../../assets/icons/curriculum-landing-started-advanced.svg#small-light");
   --curriculum-landing-started-employment: url("../../assets/icons/curriculum-landing-started-employment.svg#small-light");
   --curriculum-landing-started-educator: url("../../assets/icons/curriculum-landing-started-educator.svg#small-light");
+  --curriculum-module-mdn-resource: url("../../assets/icons/curriculum-mdn-resource.svg#light");
+  --curriculum-bullet: url("../../assets/icons/curriculum-bullet.svg#light");
+  --curriculum-about-covered: url("../../assets/icons/curriculum-about-covered.svg#light");
+  --curriculum-about-detail: url("../../assets/icons/curriculum-about-detail.svg#light");
+  --curriculum-about-educators: url("../../assets/icons/curriculum-about-educators.svg#light");
+  --curriculum-about-not: url("../../assets/icons/curriculum-about-not.svg#light");
+  --curriculum-about-students: url("../../assets/icons/curriculum-about-students.svg#light");
+  --curriculum-partner-bg: url("../../assets/curriculum-partner-bg.svg#light");
+  --curriculum-scrim-bg: url("../../assets/curriculum-scrim-bg.svg#light");
+  --curriculum-landing-arrow: url("../../assets/curriculum-landing-arrow.svg#light");
```

Comment:
It's not great that we add all these URLs as CSS variables, when light and dark values can be derived by adding #light/#dark to the path. Is there no other way to achieve this?

Reply:
I see the concern but this is out of scope of this PR.

Reply:
Created MP-1353 to track this.